### PR TITLE
For #8765: Use shared list widget in collections 

### PR DESCRIPTION
**Pull request by NotWoods**
image

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/TabInCollectionViewHolder.kt [hunk 1]
```diff
@@ -4,55 +4,53 @@
...
-            override fun getOutline(view: View, outline: Outline?) {
-                outline?.setRoundRect(
-                    0,
-                    0,
-                    view.width,
-                    view.height,
-                    FAV_ICON_BORDER_RADIUS_IN_DP.dpToFloat(view.context.resources.displayMetrics)
-                )
-            }
+        if (SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (eliserichards) [inline app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/TabInCollectionViewHolder.kt:39]: How are we handling this for v21-22?

Comment (NotWoods) [reply app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/TabInCollectionViewHolder.kt:39]: We don't show a foreground, just like other views

**Review by eliserichards — CHANGES_REQUESTED** [review]
This has some failing UI tests. Can you take a look @NotWoods?

**Comment by NotWoods** [comment]
I can't see the UI test logs :sweat_smile:

**Comment by eliserichards** [comment]
bors try

**Comment by eliserichards** [comment]
Let's try that again... everything seems to be failing with this log: 

```
0 matches
...
[taskcluster 2020-10-05 20:13:57.037Z] === Task Finished ===
[taskcluster 2020-10-05 20:13:57.116Z] Artifact "public/build/x86/target.apk" not found at "/builds/worker/checkouts/src/app/build/outputs/apk/debug/app-x86-debug.apk"
...
[taskcluster 2020-10-05 20:13:57.455Z] Unsuccessful task run with exit code: 128 completed in 26.537 seconds
```

**Comment by eliserichards** [comment]
bors try

**Review by eliserichards — APPROVED** [review]
It's all green now! ✅

**Pull request merged**